docs/doc/80-releases/95-v1.1.0.md

```diff
@@ -0,0 +1,45 @@
+---
+title: What's new in the v1.1 series release
+sidebar_label: v1.1
+description:
+  Release notes for v1.1 series of Databend.
+---
+
+Databend v1.0 was officially released on April 14, 2023! 
```

Comment:
Databend v1.0 was officially released on April 14, 2023! Thanks to all the community partners who participated and to everyone who contributed to making Databend better!